pnpm-lock.yaml

```diff
@@ -2837,6 +2840,14 @@ importers:
   components/teamdeck:
     specifiers: {}
 
+  components/teamgate:
```

Comment:
I guess `Teamgate`'s dependencies were also added here and it seems it uses `@pipedream/platform` version `0.10.0`. If possible can we omit these in this PR?